Assets/Resources/JSON/ShopItems.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
This should be recorded in a spreadsheet on Drive (and exported from there) if it's not already: https://chrome.google.com/webstore/detail/export-sheet-data/bfdcopkbamihhchdnjghdknibmcnfplk?hl=en